pcl_conversions/package.xml

```diff
@@ -15,20 +18,29 @@
   <url type="repository">https://github.com/ros-perception/perception_pcl</url>
   <url type="bugtracker">https://github.com/ros-perception/perception_pcl/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_export_depend>eigen</build_export_depend>
+  <build_depend>eigen3</build_depend>
```

Comment:
Oops this should be `eigen` please change back :-)

Reply:
You can always check here https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml